src/stats/stats.jl

```diff
@@ -58,6 +61,12 @@ function show(io :: IO, stats :: ExecutionStats)
   println("  feasibility measure: $(stats.feas_norm)")
   println("  iterations: $(stats.iter)")
   println("  elapsed time: $(stats.elapsed_time)")
+  if length(stats.solver_specific) > 0
+    println("  solver specifics:")
+    for (k,v) in stats.solver_specific
+      println("    $k: $v")
```

Comment:
Could I change all this to `show(io, ` and define `print` and `println` as aliases? Or should `show` and `print` be different?